egs/jsut/tts1/local/data_prep.sh

```diff
@@ -42,12 +35,10 @@ echo "finished making wav.scp, utt2spk, spk2utt."
 find ${db} -name "transcript_utf8.txt" | sort | while read -r filename; do
```

Comment:
```suggestion
find ${db} -follow -name "transcript_utf8.txt" | sort | while read -r filename; do
```

Reply:
Fixed. I didn't know the option. Thanks!